backend/app/db_accessors/accessor.js

```diff
@@ -0,0 +1,44 @@
+import Connection from "../db/connection.js";
```

Comment:
Great job with this section I think again just add comments explaining everything!